### ISSUE TITLE
Moving DOM Element

### ISSUE THREAD

**Issue by DizzleLizzle** [tiff/wysihtml5#291]
I'm writing a commenting system that uses wysihtml5.  When the page loads, I have a div (id=commentDiv) that contains the textarea used for the wysiwyg editor and the wysiwyg5-toolbar.  At this point, everything works fine.

When a user scrolls somewhere else in the page and hits "Reply" I am attempting to move the commentDiv to the child of another element using jquery (newCommentDiv.append($('#commentDiv')); 

Visually, everything seems to work, but the functionality does not.  I've tried re-creating a wysiwyg.Editor after the append and it still does not work.  Any ideas on what the issue is?  I am not able to click in the iframe anymore or enter text.  Any help would be appreciated.

Thanks.

**Comment by r043v** [comment]
try that :
editor.fire("change_view","composer"); // force actual editor value to be writed in the textarea
editor.destroy(); // boom
$('#commentDiv').detach().appendTo(newCommentDiv).find("textarea").val(newCommentDiv.html()); // move editor, fill his text with div one
editor = new wysihtml5.Editor( ... );

---

myself i just move the position of the editor onto my div, not his place in dom (i put it in the body) and refill it, not destroy/reinit it !

**Comment by DizzleLizzle** [comment]
I don't have a "destroy" method on my wysihtml5.Editor object.  

How are you moving the position?

**Comment by r043v** [comment]
## true no destroy, where was i see that ? sorry.

for the move, it's not the editor that i move but an absolute positionned
wrap onto him, wrap size is fixed to width / height of the target edit
element to superpose, inside child (textarea & iframe) are width / height
100%
on each live edit, i reuse the same editor/textarea, just need load/save
the textarea content on the good time

2012/12/6 DizzleLizzle notifications@github.com
